src/lib/pages/frontsite/Documents/DocumentDetails/DocumentMetadata/DocumentMetadataTabs/DocumentMetadataTabs.js

```diff
@@ -77,12 +76,27 @@ class DocumentMetadataTabs extends Component {
       panes.push({
         menuItem: 'Files',
         render: () => (
-          <Tab.Pane attached={false}>
+          <Tab.Pane>
             <DocumentLinks dividers eitems={eitems} />
           </Tab.Pane>
         ),
       });
     }
+
+    const extensions = _get(metadata, 'extensions', {});
+    if (!_isEmpty(extensions)) {
+      panes.push({
+        menuItem: 'Extensions',
```

Comment:
From the user point of view, these are not extensions, they should be rendered as normal fields. They should never see extensions...

Reply:
@ntarocco, with your suggestion, we will not put everything under one tab, but we will introduce a new tab for every `normal` field